build.psm1

```diff
@@ -347,7 +347,9 @@ function Start-PSBuild {
         Write-Warning @"
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Language Primitive Tests.Equality comparison with string and non-numeric type should not be culture sensitive
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 7
7:         $val | Should -BeTrue
```